task/01-strings-tasks.js

```diff
@@ -238,53 +245,59 @@ function encodeToRot13(str) {
  *   isString(new String('test')) => true
  */
 function isString(value) {
-    throw new Error('Not implemented');
+  if (typeof value === 'string' || value instanceof String){
```

Comment:
`typeof value === 'string' || value instanceof String` return logical result. You don't need to return `true/false` value directly.